#! /bin/sh /usr/share/dpatch/dpatch-run
## 24-add-missing-headers.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: adds missing headers

@DPATCH@
--- ACE_wrappers.orig/ace/ace.mpc
+++ ACE_wrappers/ace/ace.mpc
@@ -361,6 +361,7 @@
     Guard_T.inl
     Handle_Gobbler.inl
     Lock_Adapter_T.inl
+    Metrics_Cache_T.inl
     Refcounted_Auto_Ptr.inl
     Reverse_Lock_T.inl
     TSS_T.inl
@@ -373,6 +374,7 @@
     Array.h
     Bound_Ptr.h
     CORBA_macros.h
+    Codeset_Symbols.h
     Condition_T.h
     Default_Constants.h
     Exception_Macros.h
@@ -385,6 +387,9 @@
     Lock_Adapter_T.h
     Log_Priority.h
     Malloc_Base.h
+    Map.h
+    Metrics_Cache.h
+    Metrics_Cache_T.h
     Method_Object.h
     Memory_Pool.h
     Min_Max.h
@@ -401,6 +406,7 @@
     OS_String.h
     OS_Thread_Adapter.h
     Object_Manager_Base.h
+    Obstack.h
     Pair.h
     Proactor_Impl.h
     Reactor_Impl.h
@@ -408,17 +414,24 @@
     Refcounted_Auto_Ptr.h
     Reverse_Lock_T.h
     Select_Reactor.h
+    Service_Templates.h
     SOCK_Netlink.h
     SStringfwd.h
     Static_Object_Lock.h
     Strategies.h
     String_Base_Const.h
     Svc_Conf.h
+    Svc_Conf_Param.h
     Svc_Conf_Tokens.h
     Synch.h
     Synch_Traits.h
     TSS_T.h
+    Timer_Hash.h
+    Timer_Heap.h
+    Timer_List.h
+    Timer_Queue.h
     Timer_Queuefwd.h
+    Timer_Wheel.h
     Truncate.h
     UPIPE_Addr.h
     Value_Ptr.h
--- ACE_wrappers.orig/Kokyu/Kokyu.mpc
+++ ACE_wrappers/Kokyu/Kokyu.mpc
@@ -10,11 +10,18 @@
     Kokyu.cpp
     Default_Dispatcher_Impl.cpp
     Dispatcher_Task.cpp
+    Dispatch_Deferrer.cpp
     Kokyu_defs.cpp
   }
 
+  Header_Files {
+    kokyu_config.h
+    kokyu_export.h
+  }
+
   Template_Files {
     Kokyu_dsrt.cpp
+    DSRT_CV_Dispatcher_Impl_T.cpp
     DSRT_Direct_Dispatcher_Impl_T.cpp
     DSRT_Dispatcher_Impl_T.cpp
     DSRT_Dispatch_Item_T.cpp
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/RTEvent_Serv.mpc
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/RTEvent_Serv.mpc
@@ -84,6 +84,13 @@
     Event/EC_Thread_Flags.cpp
   }
 
+  Header_Files {
+    Event/event_serv_export.h
+    Event/EC_Defaults.h
+    Event/ECG_Defaults.h
+    Event/ECG_Address_Server_Base.h
+  }
+
   Template_Files {
     Event/EC_Busy_Lock.cpp
     Event/EC_Lifetime_Utils_T.cpp
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/RTCORBAEvent.mpc
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/RTCORBAEvent.mpc
@@ -17,6 +17,7 @@
   }
 
   Header_Files {
+    Event/rtcorba_event_export.h
   }
 
   Inline_Files {
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/RTKokyuEvent.mpc
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/RTKokyuEvent.mpc
@@ -19,6 +19,10 @@
     }
   }
 
+  Header_Files {
+    Event/rtkokyu_event_export.h
+  }
+
   Template_Files {
   }
 }
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/RTSchedEvent.mpc
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/RTSchedEvent.mpc
@@ -20,6 +20,10 @@
     }
   }
 
+  Header_Files {
+    Event/sched_event_export.h
+  }
+
   Template_Files {
   }
 }
--- ACE_wrappers.orig/TAO/orbsvcs/orbsvcs/RTEvent.mpc
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/RTEvent.mpc
@@ -26,6 +26,7 @@
 
   Header_Files {
     Event_Utilities.h
+    Event_Service_Constants.h
     RtecDefaultEventDataC.h
     RtecEventCommC.h
     RtecEventChannelAdminC.h
